# Menghitung Mean, Median, dan Modus

In [7]:
import pandas as pd

file_path = 'Data Mahasiswa.csv'
data_mahasiswa = pd.read_csv(file_path)

In [9]:
ipk_values = data_mahasiswa['IPK SEMESTER 1-6']

In [13]:
mean_ipk = ipk_values.mean()
median_ipk = ipk_values.median()
modus_ipk = ipk_values.mode()[0]

**Output Hasil**

In [17]:
print(f"Mean: {mean_ipk}")
print(f"Median: {median_ipk}")
print(f"Modus: {modus_ipk}")

Mean: 2.925333333333333
Median: 3.01
Modus: 2.5


**Berdasarkan hasil perhitungan menggunakan Python, berikut adalah nilai Mean, Median, dan Modus dari IPK Semester 1-6:**

**Mean:** 2.9253  
**Median:** 3.01  
**Modus:** 2.50  


Hasil ini sama dengan perhitungan manual yang telah dikerjakan pada **poin a​**